src/api/procedures/modifyCorporateActionsAgent.ts

```diff
@@ -26,8 +26,6 @@ export type Params = { ticker: string } & ModifyCorporateActionsAgentParams;
 
 /**
  * @hidden
- *
- * @deprecated in favor of `inviteAgent`
```

Comment:
I figured these are "internal" procedures so we don't need the deprecated comment on them.